### PR TITLE
PDJB-884: fix back links on compliance update journey change pages not going back to the cya

### PR DESCRIPTION
**Pull request by TomHanmer29**
## Ticket number

PDJB-884

## Goal of change

Fixes back links on the child pages of the cya page for compliance update journeys not going back to the cya page.

## Description of main change(s)

Modifies the update journeys to have a new step config specific to update. This is the same as the existing one but carries a state.getCyaJourneyId(step) which allows the routing back to the cya page when back is pressed.

## Anything you'd like to highlight to the reviewer?

Travis left another comment about back on multi-step change pages, I'll address that in another PR

## Checklist

Delete any that are not applicable, and add explanation below for any that are applicable but haven't been done

- [ ] Unit tests for new logic (e.g. new service methods) have been added
- [ ] Test suite has been run in full locally and is passing
- [ ] Branch has been rebased onto main and run locally, with everything working as expected (both for your new feature
  and any related functionality)
- [ ] QA instructions have been added to the ticket (particularly if this is the last PR required to complete the ticket)

### CODE REVIEW

File: src/test/kotlin/uk/gov/communities/prsdb/webapp/journeys/propertyRegistration/UpdateCheckElectricalSafetyAnswersStepConfigTests.kt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.whenever
+import uk.gov.communities.prsdb.webapp.journeys.propertyRegistration.update.electricalSafety.UpdateCheckElectricalSafetyAnswersStep
+import uk.gov.communities.prsdb.webapp.journeys.propertyRegistration.update.electricalSafety.UpdateCheckElectricalSafetyAnswersStepConfig
+import uk.gov.communities.prsdb.webapp.journeys.propertyRegistration.update.electricalSafety.UpdateElectricalSafetyJourneyState
+import uk.gov.communities.prsdb.webapp.journeys.shared.Complete
+import uk.gov.communities.prsdb.webapp.testHelpers.mockObjects.AlwaysTrueValidator
+
+@ExtendWith(MockitoExtension::class)
+class UpdateCheckElectricalSafetyAnswersStepConfigTests {
```
Comment (JasminConterioSW) [inline src/test/kotlin/uk/gov/communities/prsdb/webapp/journeys/propertyRegistration/UpdateCheckElectricalSafetyAnswersStepConfigTests.kt]: We probably don't need these (or the other step config tests) as none of those steps is doing much - we've more been adding then when there are multiple options for mode etc

**Review by JasminConterioSW — APPROVED** [review]
LGTM - We probably don't need the StepConfig tests here though?

File: src/test/kotlin/uk/gov/communities/prsdb/webapp/journeys/propertyRegistration/UpdateCheckElectricalSafetyAnswersStepConfigTests.kt (hunk 1 above)
Comment (TomHanmer29) [reply src/test/kotlin/uk/gov/communities/prsdb/webapp/journeys/propertyRegistration/UpdateCheckElectricalSafetyAnswersStepConfigTests.kt]: Yep, agree these seem redundant - will remove!
